source/extensions/filters/http/bandwidth_limit/bandwidth_limit.h

```diff
@@ -65,7 +65,7 @@ class FilterConfig : public ::Envoy::Router::RouteSpecificFilterConfig {
 
   FilterConfig(const envoy::extensions::filters::http::bandwidth_limit::v3::BandwidthLimit& config,
```

Comment:
@yanavlasov had a request that we not have required check-after-creation status checks in public constructors so after debate we've been making constructors private and having public static absl::StatusOr<std::unique_ptr<classname>> create() functions which use the protected constructor

also mild preference for creation status being the last entry just for consistency

/wait

Reply:
done. Thank you for the review!